src/libs/Common_h/controls.h

```diff
@@ -184,16 +194,11 @@ class CONTROLS
     virtual void SetMouseSensivityY(float){};
     virtual short GetDebugAsyncKeyState(int vk)
     {
-        return GetAsyncKeyState(vk);
+        return 0;
```

Comment:
Should the functionality be restored here and in GetDebugKeyState?

Reply:
CONTROLS is also abstract class, which isn't used directly. Removed this to get rid of WinAPI calls.

Reply:
Maybe make this a pure virtual function?